tests/Hedgehog.Tests/GenTests.fs

```diff
@@ -18,3 +18,16 @@ let ``dateTime creates System.DateTime instances`` count =
     |> List.distinct
     |> List.length
     =! actual.Length
+
+[<Fact>]
+let ``unicode doesn't return any surrogate`` () =
+    let actual = Gen.sample 100 100000 Gen.unicode 
+    List.filter System.Char.IsSurrogate actual =! []
```

Comment:
If you're (still) following XUnit Test Patterns, the [Assertion Method](http://xunitpatterns.com/Assertion%20Method.html) is (usually) expected/actual in tests, rather than actual/expected, so this would look like this:

```fsharp
[] =! List.filter System.Char.IsSurrogate actual
```

It's up to you though, whether you'd like to apply this suggestion or not 😃

Reply:
I agree with you. I made the mistake. I did it so.